tests/neg/i12448.scala

```diff
@@ -0,0 +1,5 @@
+class A
+object Main {
+  def mkArray[T <: A]: T#AType // error // error
```

Comment:
Here you've changed the sample code from that given in the original report #12448, which did not define `A`. Without adding that definition here, the code still crashes just as shown on the ticket.  So, it looks like no progress has been made on that fuzzball and it should not be closed.

Reply:
Right. I removed the test.